### PR TITLE
Update Descheduler docs for 4.0

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/OSDOCS-189

### CODE REVIEW

File: modules/nodes-scheduler-descheduler-config-map.adoc [hunk 1]
```diff
@@ -15,6 +15,7 @@ You should have created a desceduler policy file.
 
 Run the following command to create the configuration map:
 
+[source,bash]
```
Comment (ravisantoshgudimetla) [inline modules/nodes-scheduler-descheduler-config-map.adoc]: This is not needed.

File: modules/nodes-scheduler-descheduler-create-policy.adoc [hunk 2]
```diff
@@ -11,44 +11,78 @@ You can configure the descheduler to remove pods from nodes that violate rules d
 
 To create the needed policy:
 
-. Create a YAML file similar to the following:
+. Create a Custom Resouce Definition in a YAML file similar to the following:
```
Comment (ravisantoshgudimetla) [inline modules/nodes-scheduler-descheduler-create-policy.adoc]: When we use the operator, we don't need to create configmap.

Comment (mburke5678) [reply modules/nodes-scheduler-descheduler-create-policy.adoc]: @ravisantoshgudimetla The CRD is automatically created? What is the name of the file? 
Does the user just need to `oc edit` the file to make changes?

File: modules/nodes-scheduler-descheduler-config-map.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/nodes-scheduler-descheduler-config-map.adoc]: @ravisantoshgudimetla I removed this file from docs.

File: modules/nodes-scheduler-descheduler-create-policy.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/nodes-scheduler-descheduler-create-policy.adoc]: @ravisantoshgudimetla ^^

**Comment by vikram-redhat** [comment]
@mburke5678 I am going to merge this. Please follow up on edits required.

**Pull request merged**